.circleci/config.yml

```diff
@@ -0,0 +1,69 @@
+orbs:
+  helm: circleci/helm@1.0.0
+
+version: 2.1
+
+executors:
+  alpine:
+    docker:
+      - image: python:3.9-alpine
+
+workflows:
+  version: 2
+  default:
+    jobs:
+      - lint:
+          filters:
+            tags:
+              only: /^([0-9]+)\.([0-9]+)\.([0-9]+)$/
+            branches:
+              only: /.*/
+      - push:
+          requires:
+            - lint
+          filters:
+            tags:
+              only: /^([0-9]+)\.([0-9]+)\.([0-9]+)$/
+            branches:
+              ignore: /.*/
+
+jobs:
+  lint:
+    executor: alpine
+    steps:
+      - init
+      - run:
+          name: helm lint
+          command: helm lint
+  push:
+    executor: alpine
+    steps:
+      - init
+      - run:
+          name: Helm package
+          command: |
+            helm package .
+      - run:
+          name: Install cloudsmith CLI
+          command: |
+            pip install cloudsmith-cli
+      - run:
+          name: Push
+          command: |
+            cloudsmith push helm consensys/helm ./core-stack-worker-${CIRCLE_TAG}.tgz
+
+commands:
+  init:
+    description: "Install"
+    steps:
+      - run:
+          name: Install dependencies
+          command: |
+            apk add --no-cache curl bash git openssl openssh-client 
```

Comment:
We should be able to find these deps in a prebuilt circleci images, cached in workers, you can have a look here: https://circleci.com/docs/2.0/circleci-images/